CHANGELOG.md

```diff
@@ -1,5 +1,48 @@
 # Changelog
 
+### AMI Release v20221027
+* amazon-eks-gpu-node-1.23-v20221027
+* amazon-eks-gpu-node-1.22-v20221027
+* amazon-eks-gpu-node-1.21-v20221027
+* amazon-eks-gpu-node-1.20-v20221027
+* amazon-eks-arm64-node-1.23-v20221027
+* amazon-eks-arm64-node-1.22-v20221027
+* amazon-eks-arm64-node-1.21-v20221027
+* amazon-eks-arm64-node-1.20-v20221027
+* amazon-eks-node-1.23-v20221027
+* amazon-eks-node-1.22-v20221027
+* amazon-eks-node-1.21-v20221027
+* amazon-eks-node-1.20-v20221027
+
+[Release versions](https://docs.aws.amazon.com/eks/latest/userguide/eks-linux-ami-versions.html) for these AMIs:
+* `1.23.9-20221027`
+* `1.22.12-20221027`
+* `1.21.14-20221027`
+* `1.20.15-20221027`
+
+Binaries used to build these AMIs are published:
+* s3://amazon-eks/1.23.9/2022-07-27/
+* s3://amazon-eks/1.22.12/2022-07-27/
+* s3://amazon-eks/1.21.14/2022-07-27/
+* s3://amazon-eks/1.20.15/2022-07-27/
+
+AMI details:
+* kernel: 5.4.217-126.408.amzn2
+* dockerd: 20.10.17-1.amzn2.0.1
```

Comment:
runc, dockerd and containerd versions doesn't follow the same pattern present in the older change logs.

Reply:
This is expected. We update to this version from the old one.

Here are the commit
containerd, docker: https://github.com/awslabs/amazon-eks-ami/pull/1056/commits/b0407a915034eb0c46aba641ca6745f60e51aa9c
runc: https://github.com/awslabs/amazon-eks-ami/pull/1055/commits/1120115e6ff4f3e7d1be2ff4aa928831634309e5